src/views/CountryOperations/AppealOperationTable/index.tsx

```diff
@@ -0,0 +1,183 @@
+import {
+    useMemo,
+    useContext,
+    useState,
+} from 'react';
+import {
+    generatePath,
+} from 'react-router-dom';
+import {
+    isDefined,
+    isNotDefined,
+} from '@togglecorp/fujs';
+import RouteContext from '#contexts/route';
+import useTranslation from '#hooks/useTranslation';
+import { resolveToComponent } from '#utils/translation';
+import Container from '#components/Container';
+import {
+    useSortState,
+    SortContext,
+    getOrdering,
+} from '#components/Table/useSorting';
+import Pager from '#components/Pager';
+import Link from '#components/Link';
+import Table from '#components/Table';
+import {
+    createStringColumn,
+    createNumberColumn,
+    createDateColumn,
+    createLinkColumn,
+    createProgressColumn,
+} from '#components/Table/ColumnShortcuts';
+import {
+    type GoApiResponse,
+    useRequest,
+} from '#utils/restRequest';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type AppealTableItem = NonNullable<GoApiResponse<'/api/v2/appeal/'>['results']>[number];
+const keySelector = (appeal: AppealTableItem) => appeal.id;
```

Comment:
Let's use function instead of arrow function.
The arrow function are unnamed functions.